mon-pix/app/templates/components/user-account-update-email-validation.hbs

```diff
@@ -0,0 +1,36 @@
+<div class="user-account-update-email-validation">
+  <div class="user-account-update-email-validation__cancel-button">
+    <PixIconButton
+      @icon="times"
+      aria-label={{t 'pages.user-account.account-update-email-validation.cancel-button.aria-label'}}
+      @triggerAction={{@disableEmailValidationEditionMode}}
+    />
+  </div>
+  <h3>{{t 'pages.user-account.account-update-email-validation.title'}}</h3>
+  <form>
+    <div class="user-account-update-email-validation__input">
+      <PixInput
+         @id="newEmailValidation"
+        @label={{t 'pages.user-account.account-update-email-validation.fields.new-email.label'}}
+      />
+    </div>
+    <div class="user-account-update-email-validation__informations">
+      <p>{{t 'pages.user-account.account-update-email-validation.fields.password.security-information'}}</p>
+    </div>
+    <div class="user-account-update-email-validation__input">
+      <PixInput
+        @id="password"
+        @label={{t 'pages.user-account.account-update-email-validation.fields.password.label'}}
+      />
+    </div>
+    <div class="user-account-update-email-validation__confirm-button">
+      <PixButton
+        class="user-account-update-email-validation-actions__confirm-button"
+        @type="submit"
```

Comment:
Pourquoi de type `submit` ?

Reply:
parce qu'on va envoyer les informations de l'utilisateur et les vérifier avant d'envoyer le code non ? C'est trop précipité sur cette PR ?

Reply:
Le type par défaut (button) suffit.

Reply:
Je ne comprends pas. Pourquoi par défaut ? Si c'est parce que je suis que sur du front et qu'on a pas implem la suite, je peut comprendre mais si c'est pour une autre raison, je ne saisi pas. Si tu peut développer merci :)

Reply:
PixButton : par défaut le type est _button_

Reply:
mais il faut que le submit soit déclenché par le bouton ?

Reply:
https://developer.mozilla.org/en-US/docs/web/api/htmlformelement/submit_event

However, the SubmitEvent which is sent to indicate the form's submit action has been triggered includes a submitter property, which is the button that was invoked to trigger the submit request.

Reply:
Effectivement, si l'élément `<form>` comporte une action `onSubmit`.
Par contre, si ce n'est pas le cas, et que l'action est `trigger` par l'élément `<PixButton>`, le `type button` est suffisant.

My bad, j'ai vu que dans un commit suivant, c'est bien le `<form>` qui porte l'action. ;-)